src/main/java/io/cdap/plugin/dlp/SensitiveRecordFilter.java

```diff
@@ -334,8 +350,29 @@ public int onErrorHandling() {
       return 1;
     }
 
+    /**
+     * Gets the custom template specified by user which can be either a templateId or a full custom template path
+     *
+     * @return String of templateId or custom template path
+     */
+    public String getCustomTemplate() {
+      return Strings.isNullOrEmpty(templateId) ? customTemplatePath :
```

Comment:
Please add documentation for the new customTemplatePath config.

Reply:
PR updated.